manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/cookbook/GraphityTest.scala

```diff
@@ -60,7 +60,7 @@ See <<capabilities-capacity>> for the maximum number of relationship types.
 
 To find the activity stream for a person, just follow the linked list of the friend list, and retrieve the needed amount of activities form the respective activity list of the friends.""",
       queryText =
-        "MATCH p=(me {name: 'Jane'})-[:jane_knows*]->(friend), " +
+        "MATCH p = (me {name: 'Jane'})-[:jane_knows*]->(friend), " +
```

Comment:
Casing of the relationship type: `[:JANE_KNOWS*]`
Same in the like below: `[:HAS]`